script.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const menuBtn = document.getElementById('hamburger-menu');
+const exitMenuBtn = document.querySelector('.nav-exit-button');
+const headerUl = document.getElementById('header-ul');
+
+const portfolioLink = document.querySelector('.portfolio-link');
+const aboutLink = document.querySelector('.about-link');
+const contactLink = document.querySelector('.contact-link');
+
+const portfolioSection = document.getElementById('section-2');
+const aboutSection = document.getElementById('section-3');
+const contactSection = document.getElementById('portfolio-footer');
+
+const displayNav = function() {
```

Comment:
- [ ] I would advise you to use the arrow function instead of the function keyword, this is the best practice in the industry.

Reply:
resolved! ✔️